RadixWallet/Clients/ProfileStore/ProfileStore.swift

```diff
@@ -205,14 +211,27 @@ extension ProfileStore {
 		var bdfs = accountsRecoveredFromScanningUsingMnemonic.deviceFactorSource
 		bdfs.hint.name = name
 		bdfs.hint.model = .init(model)
+
 		let accounts = accountsRecoveredFromScanningUsingMnemonic.accounts
-		let network = Profile.Network(networkID: .mainnet, accounts: accounts, personas: [], authorizedDapps: [])
+
+		// It is important that we always create the mainnet `Profile.Network` and
+		// add it, even if `accounts` is empty, since during App launch we check
```

Comment:
feels like something is missing after `since during App launch we check`, or the combination with the next line does not sound right.